bgpd/bgp_filter.c

```diff
@@ -308,6 +318,14 @@ static void as_list_delete(struct as_list *aslist)
 	else
 		list->head = aslist->next;
 
+	cur_bp = aslist->bp_list;
+	while (cur_bp) {
+		cur_bp->bp_as_excl->exclude_aspath_acl = NULL;
+		cur_bp = cur_bp->next;
+	}
+
+
```

Comment:
Can you drop all those redundant unnecessary new lines?

Reply:
done